client/src/app.js

```diff
@@ -0,0 +1,207 @@
+import "./styles/style.css";
+import { API_KEY, BASE_URL, sources} from './variable';
+//Proxy usage to get all the Author names
+const authorNames = () => {
+    let validator = {
+        get(target, key) {
+            return key in target ? taget[key] : "No Author";
+        }
+      }
+    let authors = new Proxy({}, validator);
+}
+
+const settings = {
+  spinnerSelector: '.spinner',
+  ulSelector: '#newsArticlesList',
+  gridSection: '.newsSection',
+  channelUIListsSelector: '.thumbnail',
+  BASE_URL,
+  API_KEY
+}
+class Spinner {
```

Comment:
could you please split you file into small pieces and use import/export to make them a little bit more readable?

Reply:
I separated the classes into modules